Bio/Seq.py

```diff
@@ -1976,11 +2031,13 @@ def __init__(self, data, alphabet=Alphabet.generic_alphabet):
         if isinstance(data, str):  # TODO - What about unicode?
             self.data = array.array(self.array_indicator, data)
         elif isinstance(data, (Seq, int, float)):
-            raise TypeError("The sequence data given to a MutableSeq object "
-                            "should be a string or an array "
-                            "(not a Seq object etc)")
+            raise TypeError(
+                "The sequence data given to a MutableSeq object "
+                "should be a string or an array "
+                "(not a Seq object etc)"
+            )
```

Comment:
The last 2 strings should fit in one line.

Reply:
Yes - you have a good eye for these.